dev-lang/jimtcl/jimtcl-0.83.ebuild

```diff
@@ -1,40 +1,49 @@
 # Copyright 1999-2024 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
-EAPI="8"
+EAPI=8
+
+inherit toolchain-funcs edo
 
 DESCRIPTION="Small footprint implementation of Tcl programming language"
 HOMEPAGE="https://jim.tcl.tk/"
 SRC_URI="https://github.com/msteveb/jimtcl/archive/${PV}.tar.gz -> ${P}.tar.gz"
 
 LICENSE="LGPL-2"
 SLOT="0/${PV}"
-KEYWORDS="amd64 arm ~arm64 ~m68k ~mips ~s390 x86"
+KEYWORDS="~amd64 ~arm ~arm64 ~m68k ~mips ~s390 ~x86"
 IUSE="doc static-libs"
 
-DEPEND="
-	dev-lang/tcl:0
-"
+DEPEND="dev-lang/tcl:0"
```

Comment:
tcl in DEPEND is unusual. What is it used for?

Reply:
Check the build system? What happens if you try install jimtcl without tcl installed?